CONTRIBUTING.md

```diff
@@ -49,8 +49,8 @@ Before running the tests locally, install all the dependencies in the root of th
 npm install
 ```
 To test both samples you will need the following configured in your developer org:
-* A web application
-* A SPA application
+* [A Web application](/okta-hosted-login#prerequisites)
+* [A SPA application](https://github.com/okta/samples-js-angular/tree/master/okta-hosted-login#prerequisites)
```

Comment:
Does this **only** work w/ Angular?

This might be a great chance to show how to mix/match different front-ends with this sample backend.

Reply:
This works with any front-end. I just wanted to link a section that has instructions to setup an SPA here